checks/linux.py

```diff
@@ -0,0 +1,176 @@
+        return [LEVEL_WARNING, "PipeWire capture on X11",
+                """While technically possible, most Desktop Environments do not implement the PipeWire capture portals on X11. Your captures will therefore likely not work, i.e. you will be unable to pick a window or display, or the selected source will stay empty.<br><br>
+                We generally recommend using \"Window Capture (Xcomposite)\" on X11, as \"Display Capture (XSHM)\" can introduce bottlenecks depending on your setup.<br><br>
+                From a technical standpoint a Desktop Environment should not advertise the availability of these captures if they are not implemented. This behavior can therefore be considered a bug in your Desktop Environment."""]
+
+    return [LEVEL_INFO, "X11",
+            "If you wish to capture a window or an entire display, captures are available via Xcomposite and XSHM respectively. We generally recommend sticking to \"Window Capture (Xcomposite)\" since \"Display Capture (XSHM)\" can introduce bottlenecks depending on your setup."]
+
+
+def checkDesktopEnvironment(lines):
+    isDistroNix = search('Distribution:', lines)
+    isFlatpak = search('Flatpak Runtime:', lines)
+
+    if (len(isDistroNix) <= 0) and (len(isFlatpak) <= 0):
+        return
+
+    desktopEnvironmentLine = search('Desktop Environment:', lines)
+    desktopEnvironment = desktopEnvironmentLine[0].split()
+
+    if (len(desktopEnvironment) > 3):
+        desktopEnvironment = desktopEnvironment[3:]
+        desktopEnvironment = ' '.join(desktopEnvironment)
+    else:
+        desktopEnvironment = ''
+
+    if (len(desktopEnvironment) > 0):
+        return [LEVEL_INFO, desktopEnvironment, '']
+
+
+def checkMissingModules(lines):
+    isDistroNix = search('Distribution:', lines)
+
+    if (len(isDistroNix) <= 0):
+        return
+
+    modulesMissingList = []
+    modulesCheckList = ('obs-browser.so', 'obs-websocket.so', 'vlc-video.so')
+
+    for module in modulesCheckList:
+        if not search(module, lines):
+            modulesMissingList.append(module)
+
+    if len(modulesMissingList):
+        modulesMissingString = str(modulesMissingList)
+        modulesMissingString = modulesMissingString.replace("['", "<ul><li>")
+        modulesMissingString = modulesMissingString.replace("', '", "</li><li>")
+        modulesMissingString = modulesMissingString.replace("']", "</li></ul>")
+
+        return [LEVEL_INFO, "Missing Modules (" + str(len(modulesMissingList)) + ")",
+                """You are missing the following default modules:<br>""" + modulesMissingString]
+
+
+def checkLinuxVCam(lines):
+    isDistroNix = search('Distribution:', lines)
+    isFlatpak = search('Flatpak Runtime:', lines)
+
+    if (len(isDistroNix) <= 0) and (len(isFlatpak) <= 0):
+        return
+
+    hasV4L2Module = search('v4l2loopback not installed', lines)
+
+    if len(hasV4L2Module) > 0:
+        return [LEVEL_INFO, "VCam not available",
```

Comment:
```suggestion
        return [LEVEL_INFO, "Virtual Camera not available",
```

Spell out "Virtual Camera" unless we use "VCam" somewhere else in a user-facing message.

Reply:
Changed. I kept it short because I wasn't sure whether the OBS Bot response has any length limitations for the bullet points.